pkg/extensions/events/log_sink.go

```diff
@@ -0,0 +1,23 @@
+package events
+
+import (
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+
+	"zotregistry.dev/zot/pkg/log"
+)
+
+type logSink struct {
```

Comment:
Do we really need this? Now we have two ways to do logging?

Reply:
The intent here is to provide a sink that we can use when debugging, not to provide another logging system.

We can change the name to debugSink/nullSink/stderrSink?

Reply:
I've removed the log sink, because we can test effectively using other means.